ajax/libs/nwmatcher/package.json

```diff
@@ -22,5 +22,17 @@
       "type": "git",
       "url": "git://github.com/dperini/nwmatcher.git"
     }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/dperini/nwmatcher.git",
+    "basePath": "src/",
+    "files": [
+      "nwmatcher*.js"
+    ]
+  },
+  "requiredFile": [
```

Comment:
Oh!! Sorry for the misuse! 😱
Uh...what should I use for this field?
Could we write it down in the document?